benchmarks/build.gradle

```diff
@@ -56,22 +56,45 @@ task buildH2OExtendedJarWithBenchmarks(type: Jar, dependsOn: buildH2OExtendedJar
         attributes 'Main-Class': 'water.hadoop.h2odriver'
     }
     version = ''
-    archiveBaseName = "h2o-extended-with-benchmarks"
+    archiveBaseName = 'h2o-extended-with-benchmarks'
     zip64 true
     from "$buildDir/classes/scala/main/"
     from zipTree("$buildDir/libs/h2o-extended.jar")
 }
 
-task substituteTerraform(dependsOn: buildH2OExtendedJarWithBenchmarks) {
+task cleanTerraform(type: Delete) {
+    delete "build/terraform"
+}
+
+task copyTerraform(dependsOn: cleanTerraform) {
+    doLast {
+        copy {
+            from 'src/main/terraform'
+            include "**/*.tf"
+            into "build/terraform"
+        }
+        copy {
+            from '../templates/src/terraform/aws/modules'
+            include "network/*.tf"
+            include "emr_security/*.tf"
+            into "build/terraform/aws/modules"
+        }
+    }
+}
+
+task substituteTerraform(dependsOn: [buildH2OExtendedJarWithBenchmarks, copyTerraform]) {
     doLast {
         def tfBaseDir = "${project.buildDir.toString()}/terraform/aws/"
-        def tfScripts = ["${tfBaseDir}/variables.tf", "${tfBaseDir}/modules/emr/variables.tf", "${tfBaseDir}/modules/emr/main.tf"]
+        def tfScripts = [
+            "${tfBaseDir}/variables.tf",
+            "${tfBaseDir}/modules/emr_benchmarks_deployment/variables.tf",
+            "${tfBaseDir}/modules/emr_benchmarks_deployment/main.tf"]
         tfScripts.each { path ->
             def contents = file(path).getText('UTF-8')
             contents = contents
                 .replaceAll("SUBST_H2O_JAR_FILE", "$buildDir/libs/h2o-extended-with-benchmarks.jar")
                 .replaceAll("SUBST_PACKAGE_FILE", "$buildDir/libs/sparkling-water-benchmarks_$scalaBaseVersion-$version-all.jar")
-                .replaceAll("SUBST_SW_VERSION", version)
+                .replaceAll("SUBST_SW_VERSION", version.toString())
```

Comment:
It will working without that ;-) I just wanted to get rid of a warning in InteliJ Idea. If you prefer to have `version` without to `toString()`, i can remove that.

Reply:
👍 We can keep toString call, it's fine :)